### ISSUE TITLE
Unraid install

### ISSUE THREAD

**Issue by owerrride** [AppFlowy-IO/AppFlowy-Docs#105]
Someone tried to install on Unraid? Successfully? I ran into an error
![рпрпарапрапрапр](https://github.com/AppFlowy-IO/AppFlowy-Docs/assets/115143645/0c075bde-37f4-43e9-a266-cb99b94c5a33)

**Comment by Wh1t3Rose** [comment]
Nope. I have been trying for days! I think this app just requires thing that an actually running system would have like a display and a couple sockets. I feel like the Docker Container was an afterthought and isn't really used.

**Comment by annieappflowy** [comment]
https://github.com/AppFlowy-IO/AppFlowy/issues/3275
